pkg/controllers/networkinfo/vpcnetworkconfig_handler.go

```diff
@@ -105,33 +98,23 @@ var VPCNetworkConfigurationPredicate = predicate.Funcs{
 	},
 }
 
-func getListSize(s []string) int {
-	if s == nil {
-		return 0
-	} else {
-		return len(s)
-	}
-}
-
 func buildNetworkConfigInfo(vpcConfigCR v1alpha1.VPCNetworkConfiguration) (*commontypes.VPCNetworkConfigInfo, error) {
-	org, project, err := nsxtProjectPathToId(vpcConfigCR.Spec.NSXTProject)
+	org, project, err := nsxtProjectPathToId(vpcConfigCR.Spec.NSXProject)
 	if err != nil {
-		log.Error(err, "failed to parse nsx-t project in network config", "Project Path", vpcConfigCR.Spec.NSXTProject)
+		log.Error(err, "failed to parse nsx-t project in network config", "Project Path", vpcConfigCR.Spec.NSXProject)
```

Comment:
`nsx-t` -> NSX Project

Reply:
Done. I saw many other places use this name, so we need to rename all nsx-t to NSX, I think we can make another PR to change these names.

Reply:
https://github.com/vmware-tanzu/nsx-operator/issues/679